src/AlgAssAbsOrd/Order.jl

```diff
@@ -694,7 +694,7 @@ function pmaximal_overorder_meataxe(O::AlgAssAbsOrd{S, T}, p::Union{ZZRingElem,
   return O
 end
 
-function pmaximal_overorder_tr(O::AlgAssAbsOrd, p::Int)
+function pmaximal_overorder_tr(O::AlgAssAbsOrd, p::Union{ZZRingElem, Int})
```

Comment:
```suggestion
function pmaximal_overorder_tr(O::AlgAssAbsOrd, p::IntegerUnion)
```
Why not use `IntegerUnion`?